### ISSUE TITLE
xilem_web: Should helper functions for known attributes be added to elements?

### ISSUE THREAD

**Issue by casey** [linebender/xilem#462]
I noticed that `HtmlButtonElement` has a `disabled` helper method for setting the disabled state, but that, for example, `HtmlIFrameElement` doesn't have a `src` helper method for setting the `<iframe>` `src` attribute. Is it desired to have such helper methods? If so, I'd be happy to submit a PR.

**Comment by Philipp-M** [comment]
Yeah that's actually the plan, and I'm happy for PRs implementing this, e.g. the `HtmlImageElement` does have `src` already, currently just wrapping `Attr` is fine. Ideally this should access the DOM attributes directly of the underlying element, but I have concerns regarding the resulting binary size with the code that's necessary for this (which is why there's a commented out feature gate above those traits).
